### PR TITLE
[one-cmds] Make argument group for force_quantparam

### PR DESCRIPTION
**Pull request by jinevening**
This makes argument group for force_quantparam in one-quantize.

ONE-DCO-1.0-Signed-off-by: Hyukjin Jeong <hj1.jeong@samsung.com>

---
Related to: #7896

### CODE REVIEW

**Comment by jinevening** [comment]
This PR makes options for force_quantparam are printed separately from usual quantization options.

```
hjj@hjj-ubuntu:~/ONE$ build/release.install/bin/one-quantize -h
usage: one-quantize [-h] [-v] [-V] [-C CONFIG] [-i INPUT_PATH] [-d INPUT_DATA]
                    [-f INPUT_DATA_FORMAT] [-o OUTPUT_PATH] [-p]
                    [--input_dtype INPUT_DTYPE]
                    [--input_model_dtype INPUT_MODEL_DTYPE]
                    [--quantized_dtype QUANTIZED_DTYPE]
                    [--granularity GRANULARITY] [--input_type INPUT_TYPE]
                    [--output_type OUTPUT_TYPE]
                    [--min_percentile MIN_PERCENTILE]
                    [--max_percentile MAX_PERCENTILE] [--mode MODE]
                    [--force_quantparam] [--tensor_name TENSOR_NAME]
                    [--scale SCALE] [--zero_point ZERO_POINT]

command line tool to quantize circle model

optional arguments:
  -h, --help            show this help message and exit
  -v, --version         show program's version number and exit
  -V, --verbose         output additional information to stdout or stderr
  -C CONFIG, --config CONFIG
                        run with configuation file
  -i INPUT_PATH, --input_path INPUT_PATH
                        full filepath of the input file
  -d INPUT_DATA, --input_data INPUT_DATA
                        full filepath of the input data file. if not
                        specified, run with random input data.
  -f INPUT_DATA_FORMAT, --input_data_format INPUT_DATA_FORMAT
                        file format of input data. h5/hdf5 (default),
                        list/filelist (a text file where a file path of input
                        data is written in each line), or dir/directory (a
                        directory where input data are saved)
  -o OUTPUT_PATH, --output_path OUTPUT_PATH
                        full filepath of the output file
  -p, --generate_profile_data
                        generate profiling data

arguments for quantization:
  --input_dtype INPUT_DTYPE
                        input model data type (supported: float32,
                        default=float32). Deprecated (Use input_model_dtype)
  --input_model_dtype INPUT_MODEL_DTYPE
                        input model data type (supported: float32,
                        default=float32)
  --quantized_dtype QUANTIZED_DTYPE
                        output quantized data type (supported: uint8, int16,
                        default=uint8)
  --granularity GRANULARITY
                        quantize granularity (supported: layer, channel,
                        default=layer)
  --input_type INPUT_TYPE
                        input data type (supported: uint8, int16,
                        default=quantized_dtype)
  --output_type OUTPUT_TYPE
                        output data type (supported: uint8, int16,
                        default=quantized_dtype)
  --min_percentile MIN_PERCENTILE
                        minimum percentile (0.0~100.0, default=1.0)
  --max_percentile MAX_PERCENTILE
                        maximum percentile (0.0~100.0, default=99.0)
  --mode MODE           record mode (supported: percentile/moving_average,
                        default=percentile)

arguments for force_quantparam option:
  --force_quantparam    force_quantparam option is turned on. You can
                        overwrite quantparam (scale, zero_point) to the
                        specified tensor in the quantized model.
  --tensor_name TENSOR_NAME
                        tensor name (string)
  --scale SCALE         scale (float)
  --zero_point ZERO_POINT
                        zero point (int)
```

File: compiler/one-cmds/one-quantize [hunk 1]
```diff
@@ -102,15 +102,21 @@ def _get_parser():
...
+    force_quantparam_group = parser.add_argument_group(
+        'arguments for force_quantparam option')
+
+    force_quantparam_group.add_argument(
         '--force_quantparam',
         action='store_true',
-        help='write quantparam to the specified tensor')
-    parser.add_argument(
+        help=
+        'force_quantparam option is turned on. You can overwrite quantparam (scale, zero_point) to the specified tensor in the quantized model.'
```
Comment (mhs4670go) [inline compiler/one-cmds/one-quantize:113]: How about this?
```suggestion
        help=
        'overwrite quantparam (scale, zero_point) to the specified tensor in the quantized model.'
```

**Review by seanshpark — DISMISSED** [review]
LGTM

**Review by seanshpark — APPROVED** [review]
LGTM

**Review by mhs4670go — APPROVED** [review]
LGTM

**Comment by jinevening** [comment]
@seanshpark Could you merge this? :)

**Pull request merged**